articles/hdinsight/kafka/rest-proxy.md

```diff
@@ -92,7 +92,7 @@ This code does the following action:
 1. Fetches an OAuth token from Azure AD.
 1. Shows how to make a request to Kafka REST proxy.
 
-For more information on getting OAuth tokens in python, see [Python AuthenticationContext class](/python/api/adal/adal.authentication_context.authenticationcontext). You might see a delay while `topics` that aren't created or deleted through the Kafka REST proxy are reflected there. This delay is because of cache refresh.
+For more information on getting OAuth tokens in python, see [Python AuthenticationContext class](/python/api/adal/adal.authentication_context.authenticationcontext). You might see a delay while `topics` that aren't created or deleted through the Kafka REST proxy are reflected there. This delay is because of cache refresh. The value field of Producer API has enhanced so now, it can accept a JSON Object as well as any serialized form.
```

Comment:
Please review the wordings. The value field of Producer API has been enhanced to accept a JSON Object, in addition to the serialized string format that it already accepts.